Configurations/common.tmpl

```diff
@@ -113,7 +123,7 @@
  sub dolib {
      my $lib = shift;
      return "" if $cache{$lib};
-     unless ($disabled{shared}) {
+     unless ($disabled{shared} || $lib =~ /\.a$/) {
```

Comment:
(to pick just one instance) we're okay hardcoding ".a"?  (I guess the windows .lib is not exactly a direct analogue, but is close enough to think about.)